package-generator/model/product_package.py

```diff
@@ -25,7 +25,7 @@
 import shutil
 import errno
 import os
-from util import utils
+from util import utils_file as utils
 from model.cookbook import Cookbook
 
 PACKAGES_FOLDER = "./../murano-apps"
```

Comment:
Well, I have here two comments:
1) Why do you not define a TEMPLATE_FOLDER = 'template/PackageTemplate' and them PACKAGE_TEMPLATE_CLASS = TEMPLATE_FOLDER + /Classes/GE_name.yaml and so on.